### ISSUE TITLE
👍👍Kudos for Ditto!!

### ISSUE THREAD

**Issue by dpinncoder** [sabrogden/Ditto#447]
Not an issue, just a thanks, love Ditto comment.  Don't really know if GitLab let's you Like stuff... but wanted to thank _someone_!

**Comment by shodanx2** [comment]
This app and eartrumpet have had a huge impact on how I use computers
I find any computer without those to be just incomplete !

**Comment by dpinncoder** [comment]
noice.  i am trying out eartrumpet.  didn't know windows volume mgmt could be that exciting.  🤞

**Comment by sabrogden** [comment]
Thanks for the comment, much appreciated

**Comment by Cr8zyIvan** [comment]
I would also like to express my gratitude for Ditto 😊. Thank you!

**Comment by permakitty** [comment]
You Rock! After being issued a PC for work, I had to scramble to find a paste manager. Yours is clearly the best and I've been using your app forever. Thank you for sharing your work with all the devkitties! 😽💋

**Comment by sjvudp** [comment]
**Please don't mis-use the issue tracker for discussion like that!**

**Comment by Chealer** [comment]
Indeed, this is an ITS; please discard this ticket and use [the forums](https://sourceforge.net/p/ditto-cp/discussion/) instead.
